prime-dotnet-webapi/Services/Clients/Email/CHESClient.cs

```diff
@@ -0,0 +1,148 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Net.Mail;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+
+namespace Prime.Services
+{
+    public class CHESClient : BaseService, ICHESClient
+    {
+        private static HttpClient _client;
+
+        public CHESClient(
+            ApiDbContext context,
+            IHttpContextAccessor httpContext,
+            HttpClient httpClient)
+            : base(context, httpContext)
+        {
+            _client = httpClient ?? throw new ArgumentNullException(nameof(httpClient));
+        }
+
+        public async Task SendAsync(string from, IEnumerable<string> to, IEnumerable<string> cc, string subject, string body, IEnumerable<(string Filename, byte[] Content)> attachments)
+        {
+            var chesAttachments = new List<CHESAttachment>();
+            foreach (var attachment in attachments)
+            {
+                var chesAttachment = new CHESAttachment()
+                {
+                    content = Convert.ToBase64String(attachment.Content),
+                    contentType = "application/pdf",
+                    encoding = "base64",
+                    filename = attachment.Filename
+                };
+
+                chesAttachments.Add(chesAttachment);
+            }
+
+            var requestParams = new CHESEmailRequestParams(from, to, subject, body, chesAttachments);
+
+            var requestContent = new StringContent(JsonConvert.SerializeObject(requestParams), Encoding.UTF8, "application/json");
+
+            HttpResponseMessage response = null;
+            try
+            {
+                response = await _client.PostAsync(PrimeConstants.CHES_API_URL + "/email", requestContent);
```

Comment:
the base url of the client is set up in Startup.cs, so you can use a relative URL here rather than specify the entire URL again. Try:

_client.PostAsync("email", requestContent); 

(note: no leading slash, "/email" wont work)